results.html

```diff
@@ -28,41 +28,41 @@ <h3>ICPC</h3>
 						<div id="inner">
 							<table class="underh2">
 								<tbody>
-									<tr><th>年度</th><th>チーム名</th><th>国内予選</th><th colspan="2">アジア地区予選</th><th colspan="2">World Finals</th></tr>
+									<tr><th>年度</th><th>チーム名</th><th>国内予選</th><th colspan="2">アジア地区予選</th><th colspan="2">Play Off</th><th colspan="2">World Finals</th></tr>
```

Comment:
```suggestion
									<tr><th>年度</th><th>チーム名</th><th>国内予選</th><th colspan="2">アジア地区予選</th><th colspan="2">Playoff</th><th colspan="2">World Finals</th></tr>
```
Playoff が正しい表記っぽいです
https://icpc.iisf.or.jp/asia-pacific/asia-pacific-championship-2024/